bika/lims/browser/bika_listing.py

```diff
@@ -594,7 +594,7 @@ def _process_request(self):
                     continue
                 ##logger.info("Or: %s=%s"%(index, value))
                 if idx.meta_type in('ZCTextIndex', 'FieldIndex'):
-                    self.Or.append(MatchRegexp(index, value))
+                    self.Or.append(MatchRegexp(index, '%s*' % value))
```

Comment:
How about

> `\b`
    Matches the empty string, but only at the beginning or end of a word.